packages/core-contracts/test/contracts/mocks/proxy/ImplementationMockB.test.js

```diff
@@ -0,0 +1,23 @@
+const { ethers } = hre;
+const assert = require('assert');
+
+describe('ImplementationMockB', () => {
```

Comment:
Hmmm, I don't think we should test them directly should we?

What I had in mind is that we hit every line of code in the mocks when we use them for testing other stuff. I.e. hit the setA method in ImplementationMockB in the UUPSProxy tests.